assets/js/calypsoify.js

```diff
@@ -0,0 +1,42 @@
+( function( $ ) {
+    'use strict';
+
+    /**
+     * Record checklist task click
+     */
+    $( '.checklist__task-title a, .checklist__task-secondary a' ).click( function() {
+        var $task = $( this ).closest( '.checklist__task' )
+        var status = $task.hasClass( 'is-completed' ) ? 'complete' : 'incomplete';
+        var taskTitle = $task.data('title');
```

Comment:
We might want to add a normalized, untranslated id to the array of task fields instead and use that here (or both).

I know for the initial launch none of these tasks are translated, but in the future the task titles could be changed/translated and we'll still want to associate those events together.

Reply:
Ah good thinking!  Let me know if this looks reasonable - https://github.com/Automattic/wc-calypso-bridge/pull/133/commits/6e076c1a6b2ad3aea6cf13ba9c847b790a193f23